File: core/src/com/qtech/bubbleblaster/common/logging/old/GameLoggingFormatter.java
package com.qtech.bubbleblaster.common.logging.old;

import java.util.logging.Formatter;
import java.util.logging.LogRecord;

public class GameLoggingFormatter extends Formatter {
    @Override
    public String format(LogRecord record) {
        return "[" + record.getLoggerName() + "] " + record.getLevel().getName() + ": " + record.getMessage() + "\n";
    }
}

File: src/main/kotlin/qtech/bubbles/common/logging/old/GameLoggingFormatter.kt
package qtech.bubbles.common.logging.old

import java.util.logging.Formatter
import java.util.logging.LogRecord

class GameLoggingFormatter : Formatter() {
    override fun format(record: LogRecord): String {
        return """
            [${record.loggerName}] ${record.level.name}: ${record.message}
            
            """.trimIndent()
    }
}